### PR TITLE
Blog and announce bar for v2 release

### PR DESCRIPTION
**Pull request by tpdorsey**
Adds a draft blog post and changes to announce bar for release

## Checklist

* [ ] Unit tests for the changes exist
* [ ] Tests pass on CI and coverage remains at 100%
* [x] Documentation reflects the changes where applicable
* [ ] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)
* [x] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

Selected Reviewer: @Kludex

### CODE REVIEW

**Comment by tpdorsey** [comment]
please review

**Review by Kludex — APPROVED** [review]
🎉

File: docs/blog/pydantic-v2-final.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+</aside>
+
+---
+
+The last few months have involved a whirlwind of work, and we're finally ready to announce to official release of
+Pydantic V2!
+
+## Getting started with Pydantic V2
+
+To get started with Pydantic V2, install it from PyPI.:
```
[the remark this answers is not in the document]
Comment (tpdorsey) [reply docs/blog/pydantic-v2-final.md:21]: typo, will fix

Comment (tpdorsey) [inline docs/blog/pydantic-v2-final.md:21]:
```suggestion
To get started with Pydantic V2, install it from PyPI:
```

File: docs/blog/pydantic-v2-final.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+Pydantic V2 is compatible with Python 3.7 and above.
+
+See [the docs](/) for examples of Pydantic at work.
+
+## Migration guide
+
+If you are upgrading an existing project, you can use our extensive [migration guide](../migration.md) to understand
+what has changed.
+
+If you do encounter any issues, please [create an issue in GitHub](https://github.com/pydantic/pydantic/issues) using the `bug V2` label.
```
Comment (tpdorsey) [inline docs/blog/pydantic-v2-final.md:36]:
```suggestion
If you do encounter any issues, please [create an issue in GitHub](https://github.com/pydantic/pydantic/issues/new?assignees=&labels=bug+V2%2Cunconfirmed&projects=&template=bug-v2.yml) 
using the `bug V2` label.
```

**Review by hramezani — APPROVED** [review]

**Pull request merged**